cmds/tui.c

```diff
@@ -182,11 +185,11 @@ static const char *graph_field_names[NUM_GRAPH_FIELD] = {
 	"ADDRESS",
 };
 
-#define NUM_REPORT_FIELD 12
+#define NUM_REPORT_FIELD 10
 
 static const char *report_field_names[NUM_REPORT_FIELD] = {
-	"TOTAL TIME", "TOTAL AVG", "TOTAL MIN", "TOTAL MAX", "SELF TIME",  "SELF AVG",
-	"SELF MIN",   "SELF MAX",  "CALL",	"SIZE",	     "TOTAL STDV", "SELF STDV",
+	"TOTAL TIME", "TOTAL AVG", "TOTAL MIN", "TOTAL MAX", "SELF TIME",
+	"SELF AVG",   "SELF MIN",  "SELF MAX",	"CALL",	     "SIZE",
```

Comment:
You are deleting existing code, please rebase your code to get the update.

Reply:
I sincerely apologize for an error in my recent PR. 🙏🏻🙏🏻 
I mistakenly based my changes on an outdated version of tui.c. 

I will promptly update the PR with changes based on the current main branch version of tui.c.